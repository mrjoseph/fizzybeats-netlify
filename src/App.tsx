import React from 'react';
import SocialBar from './components/SocialBar';

const App: React.FC = () => {
  return (
    <div className="App">
      <SocialBar />
    </div>
  );
}

export default App;
