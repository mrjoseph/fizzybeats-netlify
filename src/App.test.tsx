  
import React from 'react';
import App from './App';
import renderer from 'react-test-renderer';
describe('App', () => {
    describe('snapshot', () => {
        it('renders correctly', () => {
            interface Iapp {}    
            const props:Iapp = {}
            const tree = renderer.create(<App {...props}/>).toJSON();
            expect(tree).toMatchSnapshot();
          });
    })
});
