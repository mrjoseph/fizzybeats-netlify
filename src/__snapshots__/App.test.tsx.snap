// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="MuiTypography-root MuiTypography-body1"
    style={
      Object {
        "backgroundColor": "#cfe8fc",
        "height": "100vh",
      }
    }
  >
    hello
  </div>
</div>
`;
